src/fmu/dataio/export/rms/inplace_volumes.py

```diff
@@ -172,7 +173,7 @@ def _transform_and_add_fluid_column_to_table(
                 )
 
                 # add the fluid as column entry instead
-                fluid_table[_enums.InplaceVolumes.FLUID_COLUMN] = fluid
+                fluid_table[_enums.InplaceVolumes.FLUID_COLUMN.value] = fluid
```

Comment:
enums drive me crazy sometimes..

Reply:
Funny thing is that it worked just fine. Terrible thing is that it worked just fine 😅